README.md

```diff
@@ -78,6 +78,51 @@ createAction('ADD_TODO')('Use Redux');
 
 `metaCreator` is an optional function that creates metadata for the payload. It receives the same arguments as the payload creator, but its result becomes the meta field of the resulting action. If `metaCreator` is undefined or not a function, the meta field is omitted.
 
+### `createActions(?actionsMap, ?...actionTypes)`
+
+Returns an object mapping action types to action creators. The keys of this object are camel-cased from the keys in `actionsMap` and the values of the `actionTypes` arguments; the values are the action creators.
+
+`actionsMap` is an optional object with action types as keys, and whose values can be
+
+- a function, which is the payload creator for that action
+- an array with `payload` and `meta` functions in that order, as in [`createAction`](#createactiontype-payloadcreator--identity-metacreator)
+    - `meta` is **required** in this case (otherwise use the function form above)
+
+`actionTypes` is an optional list of positional arguments that are action type strings; these action types will use the identity payload creator.
+
+```js
+const { actionOne, actionTwo, actionThree } = createActions({
+  // function form; payload creator defined inline
+  ACTION_ONE(key, value) {
+    return { [key]: value };
+  },
+  
+  // array form
+  ACTION_TWO: [
+    (first, second) => [first, second],  // payload
+    (first, second) => { first, second } // meta
```

Comment:
`{ first, second }` => `({ first, second })`

I would add a third argument or only take the second, just for clarification how this works, so beginners understand directly, that `payload` and `meta` get the same arguments.

Example:

```js
(first) => first,
(first, second) => ({ second })
```